ods_ci/tests/Tests/400__ods_dashboard/430__data_science_pipelines/434__data-science-pipelines-ui.robot

```diff
@@ -70,25 +70,7 @@ Verify User Can Create, Run and Delete A DS Pipeline From DS Project Details Pag
     ${workflow_name}=    Create Pipeline Run    name=${PIPELINE_TEST_RUN_BASENAME}
     ...    pipeline_name=${PIPELINE_TEST_NAME}     run_type=Immediate
 
-    ## TODO: fix keywords checking job status to use Pipelines > Jobs
-    ## instead of the Projec Details Page
-    SeleniumLibrary.Wait Until Page Contains       Running     timeout=10s
-    SeleniumLibrary.Wait Until Page Contains       Completed   timeout=10m
-
-    # TODO: fix following keywords
-    #    Open Data Science Project Details Page    ${PRJ_TITLE}
-    #    Wait Until Pipeline Last Run Is Started    pipeline_name=${PIPELINE_TEST_NAME}
-    #    ...    timeout=10s
-    #  Wait Until Pipeline Last Run Is Finished    pipeline_name=${PIPELINE_TEST_NAME}
-    #    Pipeline Last Run Should Be    pipeline_name=${PIPELINE_TEST_NAME}
-    #    ...    run_name=${PIPELINE_TEST_RUN_BASENAME}
-    #    Pipeline Last Run Status Should Be    pipeline_name=${PIPELINE_TEST_NAME}
-    #    ...    status=Completed
-    #    Pipeline Run Should Be Listed    name=${PIPELINE_TEST_RUN_BASENAME}
-    #    ...    pipeline_name=${PIPELINE_TEST_NAME}
-
-    #    Verify Pipeline Run Deployment Is Successful    project_title=${PRJ_TITLE}
```

Comment:
The `Verify Pipeline Run Deployment Is Successful` keyword is now unused? Maybe it should be deleted at

https://github.com/red-hat-data-services/ods-ci/blob/80fd3d9e7171b84f1badd489291abff611f92b26/ods_ci/tests/Tests/400__ods_dashboard/430__data_science_pipelines/434__data-science-pipelines-ui.robot#L155-L190

Reply:
Yeah. Will create other PR to remove outdated code. The idea of this PR is to unblock the team because we are working in parallel and those changes are dependent.